.github/workflows/issue-label-assign.yml

```diff
@@ -0,0 +1,27 @@
+name: "Set Issue Label and Assignee"
+on:
+ issues:
+    types: [opened, edited]
+ pull_request:
+    types: [opened, edited]
+ pull_request_target:
+    types: [opened, edited]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Naturalclar/issue-action@v2.0.2
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          title-or-body: 'title'
+          parameters: >
+           [
+           {"keywords":["chore", "refactor", "dependencies"], "labels": ["chore", "patch"]},
```

Comment:
Do we use any of these labels these days? We mostly just use the ee/vv-related labels and then have 'bug'. But I'm not sure if we have the "chore" or "patch" labels, for example, unless im misunderstanding?

I'm also wondering how this fits into @katjam's github cleanup :)

Reply:
So these do the semantic versioning stuff. Patch is 0.0.x, minor is 0.x.0, major is x.0.0 :) we could ditch chore tho?

Reply:
Yeah I don't think the "chore" label is necessarily required :)